### PR TITLE
[ProcessCommand] Add one line between the progress bar and Exception trace

### PR DESCRIPTION
**Pull request by carusogabriel**
Add one line between the progress bar and Exception trace.
Before:
captura de tela de 2018-01-02 03-01-54
After:
captura de tela de 2018-01-02 03-03-09

### CODE REVIEW

File: src/Console/Command/ProcessCommand.php [hunk 1]
```diff
@@ -159,8 +159,9 @@ private function processFiles(array $fileInfos): void
             try {
                 $this->processFile($fileInfo, $i);
             } catch (Throwable $throwable) {
+                $this->consoleStyle->newLine();
```
Comment (TomasVotruba) [inline src/Console/Command/ProcessCommand.php]: :+1:

File: src/Console/Command/ProcessCommand.php [hunk 2]
```diff
@@ -159,8 +159,9 @@ private function processFiles(array $fileInfos): void
...
                 throw new FileProcessingException(
-                    sprintf('Processing of "%s" file failed.', $fileInfo->getRealPath()),
+                    sprintf('Processing of "%s" file failed.', $fileInfo->getRelativePathname()),
```
Comment (TomasVotruba) [inline src/Console/Command/ProcessCommand.php]: This path should be relative to `getcwd()`, so that inludes the `tests` directory and is eaier to find.

Comment (carusogabriel) [reply src/Console/Command/ProcessCommand.php]: I tried to simplify the pathname to do not include `tests`, as we passed to the `process` argument:
```bash
rector process tests/ --level phpunit-specific-method
```

Comment (TomasVotruba) [reply src/Console/Command/ProcessCommand.php]: That is true, but user has to find file itselt and we don't want to force him to remember what was passed as argument and prepend it to each file.

It's common for CLI tools (PHPUnit, PHP CS Fixer, ECS, PHPStan) to report due to current working directory.

**Comment by TomasVotruba** [comment]
Thank you!

**Pull request merged**